config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget id="org.phidatalab.radar_armt" version="0.5.6"
+<widget id="org.phidatalab.radar_armt" version="0.5.7"
```

Comment:
is it possible to change this to `0.5.7-alpha` ? and same in apk name to 57-alpha. 
Not sure if its possible or playstore accepts this.